src/gui/editors/SongEditor.cpp

```diff
@@ -242,16 +247,22 @@ SongEditor::SongEditor( Song * song ) :
 
 
 	//Set up zooming model
-	for( float const & zoomLevel : m_zoomLevels )
+	for(const auto& zoomLevel : m_zoomXLevels)
+	{
+		m_zoomingXModel->addItem(QString("%1\%").arg(zoomLevel * 100));
+	}
+	m_zoomingXModel->setInitValue(m_zoomingXModel->findText("100%"));
+	connect(m_zoomingXModel, SIGNAL(dataChanged()), this, SLOT(zoomingChanged()));
+	connect(m_zoomingXModel, SIGNAL(dataChanged()), m_positionLine, SLOT(update()));
```

Comment:
It'd be better to avoid the SIGNAL and SLOT macros and use the newer way of calling connect.
See: https://doc.qt.io/qt-5/signalsandslots.html